### ISSUE TITLE
Is it possible to add a skip filter to avoid formatting certain nodes, without using xml:space="preserve"?

### ISSUE THREAD

**Issue by atlantamatt** [chrisbottin/xml-formatter#57]
We format XHTML and we're getting complaints that <script> contents are being formatted incorrectly. Is it possible to skip formatting using an option similar to filter(but without removing them)? I'd like to avoid pre-modifying the source to add the xml:space command then remove it after.

**Comment by chrisbottin** [comment]
Hi @atlantamatt, Thanks for submitting your issue.
I see what you mean, I will have a think to see what the best option is.

**Comment by chrisbottin** [comment]
@atlantamatt I have something working like this...

```js
const xmlFormat = require('xml-formatter');
xmlFormat('<<xml content>>', {collapseContent: true,  ignoredPaths: ['/html/head/script']});
``` 
## Example 
### Input
```xml
<html>
<head>
<title>Page Title</title>
<script language="JavaScript">
    function log(msg) {
        console.log(msg);
    }
</script>
</head>
<body>
<p>

    Lorem ipsum dolor sit amet.

</p>
<p>Vivamus <b>ullamcorper</b> cursus <i>ullamcorper</i>.</p>
</body>
</html>
```

### Output

```xml
<html>
    <head>
        <title>Page Title</title>
        <script language="JavaScript">
    function log(msg) {
        console.log(msg);
    }
        </script>
    </head>
    <body>
        <p>Lorem ipsum dolor sit amet.</p>
        <p>Vivamus <b>ullamcorper</b> cursus <i>ullamcorper</i>.</p>
    </body>
</html>
```

Is it what you would want?

**Comment by atlantamatt** [comment]
I think that should work. Does it support any XPath selector?

**Comment by chrisbottin** [comment]
Hi, sorry for the long delay, I will try to submit a new version next weekend with a fix. I think initially it will support simple xpaths, no attribute support.

**Comment by chrisbottin** [comment]
New option `ignoredPaths` available in version `3.4.0`

**Comment by atlantamatt** [comment]
Thanks, I'll try it out soon.
